test-case/check-pause-release-suspend-resume.sh

```diff
@@ -75,21 +71,30 @@ OPT_NAME['l']='loop'         OPT_DESC['l']='loop count'
 OPT_HAS_ARG['l']=1             OPT_VAL['l']=5
 
 OPT_NAME['i']='sleep-period' OPT_DESC['i']='sleep period of aplay, unit is ms'
-OPT_HAS_ARG['i']=1             OPT_VAL['i']='100'
+OPT_HAS_ARG['i']=1             OPT_VAL['i']='500'
 
 OPT_NAME['s']='sof-logger'   OPT_DESC['s']="Open sof-logger trace the data will store at $LOG_ROOT"
 OPT_HAS_ARG['s']=0             OPT_VAL['s']=1
 
+OPT_NAME['t']='tplg'         OPT_DESC['t']='tplg path, default is environment variable: TPLG'
+OPT_HAS_ARG['t']=1             OPT_VAL['t']="$TPLG"
+
+OPT_NAME['T']='type'         OPT_DESC['T']="suspend/resume type from /sys/power/mem_sleep"
+OPT_HAS_ARG['T']=1             OPT_VAL['T']=""
+
 func_opt_parse_option "$@"
 
-pcm=${OPT_VAL['p']}
-fmt=${OPT_VAL['f']}
-channel=${OPT_VAL['c']}
-rate=${OPT_VAL['r']}
 repeat_count=${OPT_VAL['l']}
 sleep_period=${OPT_VAL['i']}
 test_mode=${OPT_VAL['m']}
 file_name=${OPT_VAL['F']}
+tplg=${OPT_VAL['t']}
+
+if [ "${OPT_VAL['T']}" ]; then
```

Comment:
```suggestion
if [ -n "${OPT_VAL['T']}" ]; then
```
otherwise an incorrect `"${OPT_VAL['T']}"` could generate very confusing behaviors.